openreview/workflows/workflows.py

```diff
@@ -0,0 +1,938 @@
+                                'type': 'integer'
+                            }
+                        }
+                    },
+                    'noteId': {
+                        'value': {
+                            'param': {
+                                'type': 'string' 
+                            }
+                        }
+                    }
+                },
+                'replacement': True,
+                'invitation': {
+                    'id': f'{support_group_id}/Venue_Request/Reviewers_Only' + '${2/content/noteNumber/value}' + '/-/Deployment',
+                    'signatures': [self.super_id],
+                    'readers': ['everyone'],
+                    'writers': [support_group_id],
+                    'invitees': [support_group_id],
+                    'maxReplies': 1,
+                    'process': '''def process(client, edit, invitation):
+    meta_invitation = client.get_invitation(invitation.invitations[0])
+    script = meta_invitation.content['deploy_process_script']['value']
+    funcs = {
+        'openreview': openreview,
+        'datetime': datetime
+    }
+    exec(script, funcs)
+    funcs['process'](client, edit, invitation)
+''',
+                    'edit': {
+                        'signatures': { 
+                            'param': { 
+                                'items': [ { 'value': support_group_id, 'optional': True } ] 
+                            }
+                        },
+                        'readers': ['${{2/note/id}/readers}'],
+                        'writers': [support_group_id],
+                        'note': {
+                            'id': '${4/content/noteId/value}',
+                            'forum': '${4/content/noteId/value}',
+                            'ddate': {
+                                'param': {
+                                    'range': [ 0, 9999999999999 ],
+                                    'optional': True,
+                                    'deletable': True                                 
+                                }
+                            },
+                            'signatures': ['${3/signatures}'],
+                            'content': {
+                                'venue_id': {
+                                    'value': {
+                                        'param': {
+                                            'type': 'string',
+                                            'regex': '.*'
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        )
+
+        self.post_invitation_edit(invitation)
+
+    def set_reviewers_only_comment(self):
+
+        support_group_id = self.support_group_id
+        comment_invitation_id = f'{support_group_id}/Venue_Request/Reviewers_Only/-/Comment'
```

Comment:
this invitation seems to be very similar to all the workflow requests, can we use one invitation for all requests? same as the deployment invitation.

Reply:
I am not sure about using the same deployment invitation, because each workflow will have its own deployment process. 
I can make the change for the comment invitation now.

Reply:
I suggest using the same code to create the invitation and parametrie the name of the process function?